### ISSUE TITLE
Change barycentering position in prepfold?

### ISSUE THREAD

**Issue by dlakaplan** [scottransom/presto#148]
Is it possible to convince `prepfold` to barycenter to a position other than the RA/Dec in the header?  With big beams there can be barycentering errors from observations done before gridding.  This doesn't make a big difference for proper timing but before I have a solution I am trying to just compare P/Pdot values.  Still probably not large in this particular case but I was wondering if it's possible.

(Alternative is to just edit the header)

**Comment by scottransom** [comment]
Yeah, there isn't any way to do that right now besides editing the header.
